### ISSUE TITLE
Module "chai" not found

### ISSUE THREAD

**Issue by Aratramba** [TheSavior/find-parent#13]
installed it through npm and got this error:

`module "chai" not found from ".../node_modules/find-parent/index.js"`. 

I see chai is being used for `assert.isObject(opts);` but that could probably better be replaced by something like https://www.npmjs.com/package/is-object

**Comment by elicwhite** [comment]
Or even just `typeof(opts) === 'object'`. I'd happily accept a PR removing the dependency.

**Comment by Aratramba** [comment]
i don't get it. Tests keep complaining about the error.

I throw the error:

``` js
if (typeof opts !== 'object' || opts === null) { 
    throw new Error('Expected opts to be an object.');
}
```

But the tests don't seem to think so:

`find-parent/index #byClassName should throw if opts is defined and not an object FAILED`

Even if I run the thing in the browser, like `FindParent.byClassName(document, 'foo', function(){})`, it throws an error. What am I doing wrong?

**Comment by elicwhite** [comment]
Is the failure that it expected it to throw and it didn't or that it threw with a different message. If you put up a PR I can check out the log.

**Comment by elicwhite** [comment]
Your PR is merged and released as 2.0.2. Sorry that it took so long.

**Comment by Aratramba** [comment]
great, thanks!
